.github/workflows/ci.yml

```diff
@@ -84,39 +160,38 @@ jobs:
       contents: read  # This is required for actions/checkout
     needs:
       - ci-vars
-      - build
+      - build-windows
     secrets: inherit
     uses: ./.github/workflows/test-wheel-windows.yml
     with:
       build-type: pull-request
       host-platform: ${{ matrix.host-platform }}
       build-ctk-ver: ${{ needs.ci-vars.outputs.CUDA_BUILD_VER }}
-      matrix_filter: "map(select([.CUDA_VER // empty | split(\".\")[] | tonumber] as $v | ($v[0] < 13)))"
 
-#  doc:
-#    name: Docs
-#    if: ${{ github.repository_owner == 'nvidia' }}
-#    # Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
-#    permissions:
-#      id-token: write
-#      contents: write
-#      pull-requests: write
-#    needs:
-#      - ci-vars
-#      - build
-#    secrets: inherit
-#    uses: ./.github/workflows/build-docs.yml
-#    with:
-#      build-ctk-ver: ${{ needs.ci-vars.outputs.CUDA_BUILD_VER }}
+  doc:
```

Comment:
Please disable doc builds because it does not work in the bp branch: https://github.com/NVIDIA/cuda-python/issues/809.